static/app.js

```diff
@@ -26,5 +27,34 @@ async function get_random_game() {
         });
 }
 
+async function last_username(){
+      fetch('last_username')
+            .then(async response => {
+                if (response.ok) {
+                    document.getElementById("user").value = await response.text();
+                } else {
+                    const error_field = document.getElementById("user");
+                    error_field.innerText = await response.text();
+                }
+            });
+}
+
+async function page_visits(){
+      fetch('page_visit_counter')
+        .then(async response => {
+            if (response.ok) {
+                const page_visits = await response.text();
+                const page_visits_field = document.getElementById("page-visits");
+                page_visits_field.innerHTML += page_visits;
```

Comment:
Is there a reason you're appending to the page-visits paragraph instead of overwriting the value?

Reply:
Ohhhhh yeah I can see this being a problem